### ISSUE TITLE
OLW will not insert a split post

### ISSUE THREAD

**Issue by ruthross** [OpenLiveWriter/OpenLiveWriter#633]
Goody goody. Another thing the updated OLW CANNOT DO. I cannot insert a **split post**. What gives. These are such simple things. Can't you guys FIX THEM? Very frustrated. New OLW stinks.

**Comment by ScottIsAFool** [comment]
Thanks for reporting this. The split post functionality is still present, it's just for some reason the visual indicator isn't showing, we are investigating that. If you leave a gap between the post details where you want the split to appear (you need to do that first), then press the split post button, you'll find that the functionality should still work.

Closing this issue though as it is a duplicate of #629

**Comment by ruthross** [comment]
Still doesn't work despite my doing what you said.

Ruth Ross

**Comment by jongalloway** [comment]
@ruthross I built an installer with the previous version that makes it easy to disable automatic updates. You can install it from here: https://olw.blob.core.windows.net/sunset/OpenLiveWriterSetup.exe

**Comment by ruthross** [comment]
Thanks, Jon. I am still having trouble with the lack of Table Tools. When I import from Constant Contact or Mail Chimp, I cannot format to my liking because of columns and rows that I no longer see but are still there. I think I have to cut and paste piecemeal, not the entire email.

 

 

Ruth Ross

www.njartsmaven.com

njartsmaven@gmail.com

**Comment by ruthross** [comment]
Jon, I installed your fixer to disable automatic updates and now I cannot publish! I get an error message. And, my shortcut to Open Live Writer doesn't work. What the hell happened? Can you help me?

 

 

Ruth Ross

www.njartsmaven.com

njartsmaven@gmail.com

**Comment by ruthross** [comment]
Jon, 

 

I installed your fix and it screwed up my access to Open Live Writer. In fact, the program and a shortcut disappeared. 

 

When I went back to Open Live Writer and reinstalled the program, I got the following error message when I tried to publish:

 

The service blogger has thrown an exception: Google.GoogleApiException: Google.Apis.Requests.RequestError

We're sorry, but you don't have permission to access this resource. [403]

Errors [

               Message[We're sorry, but you don't have permission to access this resource.] Location[ - ] Reason[forbidden] Domain[global]

]

 

   at Google.Apis.Requests.ClientServiceRequest`1.Execute()

   at OpenLiveWriter.BlogClient.Clients.GoogleBloggerv3Client.NewPost(String blogId, BlogPost post, INewCategoryContext newCategoryContext, Boolean publish, String& etag, XmlDocument& remotePost)

   at OpenLiveWriter.BlogClient.Blog.NewPost(BlogPost post, INewCategoryContext newCategoryContext, Boolean publish)

   at OpenLiveWriter.PostEditor.UpdateWeblogAsyncOperation.DoWork()

   at OpenLiveWriter.CoreServices.AsyncOperation.InternalStart()

 

I have no idea what happened. Please help me get OLW up and working again. Otherwise, I am out of business.

 

Thank you,

 

Ruth Ross

www.njartsmaven.com

njartsmaven@gmail.com

**Comment by jongalloway** [comment]
Thank you for posting the detailed error message. First, to get you at least able to publish again, can you run the standard installer from here? http://openlivewriter.org/

**Comment by ScottIsAFool** [comment]
@ruthross before you reinstall 0.6.2 as @jongalloway has asked, could you try re-adding your Blogger account to the one you have already installed? This should then allow you to post.

**Comment by ruthross** [comment]
Scott,

 

Well, I did it. And it worked. I posted a notice that says TEST. When I went to sign in to the www.njartsmaven.com blog, I was welcomed and it said a new blog. I can't sign in to that one!

This is a debacle.

 

Ruth Ross

www.njartsmaven.com

njartsmaven@gmail.com

**Comment by ScottIsAFool** [comment]
@ruthross Sorry, that last part makes no sense, you said it posted ok, but then you can't sign in?!

**Comment by ruthross** [comment]
Evidently, Blogger is not recognizing my old blog and is asking me to start from scratch to create a new blog. The 3 test posts appeared on the www.njartsmaven.com site, but when I went to sign in so I could delete them, I can't. I signed out, then signed in again but could not log in with my email and password. I got this notice: We are sorry, but you do not have access to this service. Please log in to <http://admin.google.com/> admin.google.com to enable services.  <http://www.google.com/support/a/answer/182442?hl=en> Learn more

 

I am lost in the weeds. I really don't want to have to set up this blog again. I did it 7 years ago and have weathered the loss of Windows Live Writer and the vicissitudes of Open Live Writer. I am at a loss. I even did what Jon Galloway suggested, but no go.

 

Help.

 

Thank you,

 

 

Ruth Ross

www.njartsmaven.com

njartsmaven@gmail.com

**Comment by ScottIsAFool** [comment]
Unfortunately, this sounds like something you need to take up with Google. In terms of the test post, you should be able to delete that through Live Writer (I think). If you click on file > Open Recent Post > Click on your blog > click on the blog post to delete > hit delete.

**Comment by ruthross** [comment]
Scott, I was able to delete those test posts. I tried to post with the old njartsmaven blog, but got that error message again. I have no idea how to get to Google to resolve this. Thanks for your help. I have to say that the update to OLW, while probably unavoidable, has caused me untold grief. I am at my wit's end.

 

Ruth Ross

www.njartsmaven.com

njartsmaven@gmail.com

**Comment by ScottIsAFool** [comment]
Let me see if I can work out what you're saying. Are you saying that you now have two blogs in OLW, both for the same blog, one lets you post, the other doesn't? If so, just delete the blog that is no longer working, you are safe to do that. You can do that by clicking on  File > Options > Accounts, then selecting the old blog (likely the first of the two) and clicking Remove. 

As for contacting Google, you'd have to go to the Blogger Google Groups page (if what I've described above is not what you're describing).

**Comment by ruthross** [comment]
Jon, OK I did what you and Scott Lovegrove suggested. I am able to post on the new blog account. Accessing it through Google and Blogger is another problem that I'll have to solve.

 

Thanks for your help.

 

 

Ruth Ross

www.njartsmaven.com

njartsmaven@gmail.com

**Comment by ruthross** [comment]
Scott,

I did what you suggested and was able to post on the new blog account. I usually sign in to the blog via blogger to delete a post, but you have shown me how to do it from OLW. So I take back my kvetching about the new version of OLW. I think I got bollixed up with Jon Galloway's update, which I probably didn't need. You have been most helpful. I will follow your directions and delete the no longer working account for njartsmaven from OLW so I don't get confused. BTW, charmcityartsmaven blog published with nary an error message.

 

Thanks again for the Google advice.

 

Best,

 

Ruth Ross

www.njartsmaven.com

njartsmaven@gmail.com
